src/RemoteSymbolProvider/StadiaSymbolStoreSymbolProvider.cpp

```diff
@@ -35,26 +37,29 @@ Future<ErrorMessageOr<CanceledOr<std::filesystem::path>>>
 StadiaSymbolStoreSymbolProvider::RetrieveSymbols(
     const orbit_symbol_provider::ModuleIdentifier& module_id, orbit_base::StopToken stop_token) {
   std::filesystem::path module_path(module_id.file_path);
-  std::vector<orbit_ggp::Client::SymbolDownloadQuery> queries = {
-      {module_path.filename().string(), module_id.build_id}};
+  orbit_ggp::Client::SymbolDownloadQuery query = {module_path.filename().string(),
+                                                  module_id.build_id};
 
   // TODO(b/245920841): Client::GetSymbolDownloadInfoAsync should support returning ErrorMessage and
   // NotFound.
-  return orbit_base::UnwrapFuture(ggp_client_->GetSymbolDownloadInfoAsync(queries).ThenIfSuccess(
+  return orbit_base::UnwrapFuture(ggp_client_->GetSymbolDownloadInfoAsync(query).ThenIfSuccess(
       main_thread_executor_.get(),
       [this, module_file_path = module_id.file_path, stop_token = std::move(stop_token)](
-          const std::vector<orbit_ggp::SymbolDownloadInfo>& download_info) mutable
+          const NotFoundOr<SymbolDownloadInfo>& call_ggp_result) mutable
       -> Future<ErrorMessageOr<CanceledOr<std::filesystem::path>>> {
-        if (download_info.empty()) return ErrorMessage{"No symbol download info returned."};
+        if (orbit_base::IsNotFound(call_ggp_result)) {
+          return ErrorMessage{"Symbols not found in Stadia symbol store"};
+        }
```

Comment:
I would add a TODO bug here for: Change to return NotFound as soon as SymbolProvider supports SymbolLoadingOutcom

Reply:
I have a follow up PR ready for using `SymbolLoadingOutcome` 😂